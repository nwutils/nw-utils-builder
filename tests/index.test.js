const nwBuilder = require('../src/index.js');

describe('nw-utils-builder', () => {
  test('Global default settings', () => {
    expect(nwBuilder.settings)
      .toMatchSnapshot();
  });
});
