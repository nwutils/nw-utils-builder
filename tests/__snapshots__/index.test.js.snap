// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nw-utils-builder dryRun Settings are empty 1`] = `
Object {
  "options": Object {
    "concurrent": true,
    "mirror": "https://dl.nwjs.io/",
    "output": "./dist",
    "verbose": true,
  },
  "taskDefaults": Object {
    "arch": "x86",
    "excludes": Array [],
    "files": Array [
      "**/*",
    ],
    "icon": undefined,
    "junk": Array [],
    "manifestOverrides": Object {},
    "nwFlavor": "normal",
    "nwVersion": "match",
    "outputPattern": "{{name}}-{{version}}-{{platform}}-{{arch}}",
    "outputType": "zip",
    "platform": "win",
    "strippedManifestProperties": Array [],
    "unIcon": undefined,
  },
  "tasks": Array [
    Object {
      "arch": "x86",
      "excludes": Array [],
      "files": Array [
        "**/*",
      ],
      "icon": undefined,
      "junk": Array [],
      "manifestOverrides": Object {},
      "nwFlavor": "normal",
      "nwVersion": "match",
      "outputPattern": "{{name}}-{{version}}-{{platform}}-{{arch}}",
      "outputType": "zip",
      "platform": "win",
      "strippedManifestProperties": Array [],
      "unIcon": undefined,
    },
  ],
}
`;
